pkg/rhcos/powervs_regions.go

```diff
@@ -0,0 +1,70 @@
+package rhcos
+
+// TODO(cklokman): This entire file should be programatically generated similar to aws
+
+// PowerVSZones holds the zones cooresponding to each region found in PowerVS Regions
+// TODO(cklokman): These are fictional zones for testing
+var PowerVSZones = map[string][]string{
+	"eu-de": []string{
+		"eu-de-1",
+		"eu-de-2",
+	},
+	"dal": []string{"dal12"},
+	"lon": []string{
+		"lon04",
+		"lon06",
+	},
+	"mon":     []string{"mon01"},
+	"osa":     []string{"osa21"},
+	"syd":     []string{"syd04"},
+	"sao":     []string{"sao01"},
+	"tor":     []string{"tor01"},
+	"tok":     []string{"tok04"},
+	"us-east": []string{"us-east"},
+}
+
+// PowerVSRegions holds the regions for IBM Power VS, and descriptions used during the survey
+// TODO(cklokman): These are fictional regtions for testing
+var PowerVSRegions = []map[string]string{
```

Comment:
testing regions should probably go under the _test.go files

Reply:
ah, misleading comment there. testing in this context should have been _development_. that TODO needs to removed and these zones need to be triple-checked. i missed a few stray TODOs there we should clean up -- including the one about programmatic generation (since we don't have a way to query zones)